HHS/Head-Start-TTADP
# [TTAHUB-2121] Group share front end

## Description of change

This change the mockup on [TTAHUB-2121](https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2121) and can be found here:

[UX Design](https://www.figma.com/proto/mZON6YeDBYnwKWHEXkpN8j/My-Groups---share-groups?page-id=0%3A1&type=design&node-id=896-19956&viewport=1374%2C1862%2C0.16&t=d3IXrBJw6FhmS92V-1&scaling=min-zoom&starting-point-node-id=896%3A19956&show-proto-sidebar=1&mode=design)

The change modifies the default behavior of the groups page to allow for sharing of editing (**co-owners**) and visibility (**individuals**). ATM there is no BE for this page so you will need to fake out the request responses to reproduce page behavior.

## How to test

- Review the code
- All tests pass
- Mock data response for co-owners and individuals (MyGroups.js line 141)
- Make sure that anytime we change the selected recipients we refresh both the selected and available users (co-owners/individuals).

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2121


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
